x/ecocredit/client/tx.go

```diff
@@ -318,7 +319,39 @@ Parameters:
 // to another.
 func TxSendCmd() *cobra.Command {
 	return txFlags(&cobra.Command{
-		Use:   "send [recipient] [credits]",
+		Use:   "send [amount] [batch_denom] [recipient]",
+		Short: "Sends credits from the transaction author (--from) to the recipient",
+		Long: `Sends credits from the transaction author (--from) to the recipient.
+
+Parameters:
+  amount: amount to send
+  batch_denom: batch denomination
+  recipient: recipient address`,
+		Args: cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := sdkclient.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+			credit := core.MsgSend_SendCredits{
+				TradableAmount: args[0],
+				BatchDenom:     args[1],
```

Comment:
I think it would be nice to add an argument for retirement amount as well. You can use an optional flag for retirement jurisdiction. The retirement jurisdiction flag doesn't need to be required in case retired amount is `0`.

`GenerateOrBroadcastTxCLI` calls a message validation function so the command will error if retirement jurisdiction is not set and retired amount is positive, so not need to worry about that logic within the command logic.